src/screens/EPub/views/EditEPubStructure/QuickActions.js

```diff
@@ -83,7 +93,27 @@ function QuickActions({ chapters = {}, items, currChIndex = 0, dispatch }) {
           </Button>
         } */}
       </ButtonGroup>
+      <CTFragment dFlexCol>
+        <form onSubmit={handleOnSubmit}> 
+          <TextField
```

Comment:
I think default behavior should be having the text box blank and if a user does not enter a value it will use some default value (say 30)